lib/internal/policy/manifest.js

```diff
@@ -41,6 +41,8 @@ const shouldAbortOnUncaughtException =
   getOptionValue('--abort-on-uncaught-exception');
 const { abort, exit, _rawDebug } = process;
 
+const TERMINATE = () => null;
```

Comment:
Nit... just to match style elsewhere .... but can be done later
```suggestion
const kTerminate = () => null;
```